pkg/api/customization/monitor/handler_utils.go

```diff
@@ -63,40 +80,102 @@ func (p *clusterGraphInputParser) parseFilter() error {
 		return fmt.Errorf("clusterId is empty")
 	}
 
+	p.ResourceType = p.Input.Filters["resourceType"]
+
+	if p.ResourceType == "" {
+		p.ResourceType = p.Input.Filters["displayResourceType"]
+	}
+
+	if p.ResourceType == "" {
+		return fmt.Errorf("resourceType is empty")
```

Comment:
Will be good to specify the exact path to the missing parameter in the error, like Filters["resourceType"]